mysql/helm/README.md

```diff
@@ -0,0 +1,20 @@
+# MYSQL
+
+### How to install
+
+Make sure you have a lokomotive cluster up and running with [pushgateway](https://github.com/kinvolk/service-mesh-benchmark/tree/master/configs/pushgateway) installed.
+
+```bash
+helm install \
+  --create-namespace \
+  --namespace mysql \
+  mysql .
+```
+
+### How to uninstall
+
+```bash
+helm uninstall mysql -n mysql
+
+kubectl delete ns mysql
+```
```

Comment:
Nit: I think it's worth adding build instructions for the container image, and noting that the `Dockerfile` requires `benchmark-base` (which in turn requires `dstat-builder`).

Reply:
Added the build target to the Makefile itself. Added a README to the `mysql` directory, since this directory only deals with the helm chart.